src/component/admin/groups/GroupsList/GroupCard/GroupCard.tsx

```diff
@@ -20,9 +20,21 @@ const StyledGroupCard = styled('aside')(({ theme }) => ({
     border: `1px solid ${theme.palette.dividerAlternative}`,
     borderRadius: theme.shape.borderRadiusLarge,
     boxShadow: theme.boxShadows.card,
+    display: 'flex',
+    flexDirection: 'column',
     [theme.breakpoints.up('md')]: {
         padding: theme.spacing(4),
     },
+    '&:hover': {
+        transition: 'background-color 0.2s ease-in-out',
+        backgroundColor: theme.palette.neutral.light,
+    },
+}));
+
+const StyledTitleRow = styled('div')(() => ({
```

Comment:
Maybe this could also extend `StyledRow` (like StyledBottomRow is doing) instead of `div`? Only thing that changes is `alignItems`.

Reply:
Good one. Changed.